examples/nextjs-spa/src/pages/settings.tsx

```diff
@@ -20,55 +19,50 @@ import {
   UserSettingsCard,
   UserSettingsFlowType,
 } from "@ory/elements"
-import { HandleError } from "../pkg/hooks"
+import { HandleError } from "@/pkg/hooks"
 import { NextPageWithLayout } from "./_app"
 
 const Settings: NextPageWithLayout = () => {
   const [flow, setFlow] = useState<SettingsFlow>()
-  const handleError = HandleError()
 
   // Get flow information from the URL
   const router = useRouter()
 
   const flowId = String(router.query.flow || "")
   const returnTo = String(router.query.return_to || "")
 
-  const getSettingsFlow = useCallback(
+  const getFlow = useCallback(
     (id: string) =>
       ory
         .getSettingsFlow({ id })
         .then(({ data }) => {
           setFlow(data)
         })
-        .catch((err: AxiosError) => {
-          if (err.response?.status === 401) {
-            router.push("/login")
-          }
-          return err
-        })
-        .catch((err: AxiosError) => handleError(err)),
-    [handleError, router],
+        .catch(handleError),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I'm not sure if we should disable linter rules like this.
Since the functions are memoized through the `useCallback` function, it should be alright to add them to this dependency list.